### ISSUE TITLE
In desktop width, the profile dropdown is positioned in the wrong scrolling context

### ISSUE THREAD

**Issue by Aesthetikx** [mastodon/mastodon#20966]
### Steps to reproduce the problem

1. Use a wide monitor to view the wide interface
2. Login
3. Click the vertical ellipsis menu to open the profile drop down
4. Scroll

### Expected behaviour

The dropdown remains positioned below the ellipsis

### Actual behaviour

The dropdown scrolls with the center panel content

### Detailed description

![dropdown](https://user-images.githubusercontent.com/1302406/202560216-445fdc2e-cbc9-4c83-a968-56a7f78f1e6f.gif)

### Specifications

Mastodon v4.0.2
Firefox

**Comment by afontenot** [comment]
This is true of *every* menu on the screen, including e.g. the emoji picker. I'm really not sure why we don't just set `position: fixed` on menus. I don't think it ever makes sense to scroll them. Maybe it makes sense to automatically close some of them if the user starts scrolling, but scrolling them is weird behavior.

**Comment by Aesthetikx** [comment]
> This is true of _every_ menu on the screen, including e.g. the emoji picker. I'm really not sure why we don't just set `position: fixed` on menus. I don't think it ever makes sense to scroll them. Maybe it makes sense to automatically close some of them if the user starts scrolling, but scrolling them is weird behavior.

I think it makes sense / is fine for for the ones that are in the scrolling area, e.g. the dropdown scrolls but so does the tripple dot menu -- just not for this one (maybe others, not sure) because the tripple dot is position sticky / fixed in the viewport, so the dropdown scrolls away from the tripple dot.

**Comment by 3quarterscolon** [comment]
This is an annoyance on the profile page, but on live-updating pages like the explore page, it makes parts of the site much harder to use.

**Comment by ClearlyClaire** [comment]
Related to #14266. Tempted to close as duplicate because it's the same root issue, but leaving the issue for now as it manifests a bit differently.
